neo/Prompt/Commands/WalletImport.py

```diff
@@ -181,7 +181,7 @@ def execute(self, arguments):
 
         # validate minimum required signing key count
         signing_keys = arguments[2:]
-        len_signing_keys = len(signing_keys)
+        len_signing_keys = len(signing_keys) + 1  # since own pub key will be a signing key
```

Comment:
:thumbsup:
Alternatively we could have moved this line up
https://github.com/CityOfZion/neo-python/blob/e561e5b57d4093a9ee0c87a8312d4a4d16f6fd5e/neo/Prompt/Commands/WalletImport.py#L197
and in between 
```
    signing_keys = arguments[2:]
    len_signing_keys = len(signing_keys)
```

Reply:
Oh, I think this makes more sense! Thank you for the suggestion! Then I don't have to comment on something that will happen in the future.